service/service.go

```diff
@@ -43,7 +44,14 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 	if err != nil {
 		log.Fatal(ctx, "failed to initialise mongo db client", err)
 		return nil, err
-	} 
+	}
+
+	// Get RDS client
+	rds, err := serviceList.getRDSDB(ctx, cfg)
```

Comment:
We're replicating work here - a new PGX Pool is being opened at line `57` below - complete with external service list initialisation also. See here: https://github.com/ONSdigital/dp-areas-api/blob/6a6a7a395422eb9ec790b9deaba2dd24dcfa3220/pgx/pgx.go#L33